Makefile

```diff
@@ -308,7 +308,19 @@ watch-frontend: node-check $(FOMANTIC_DEST) node_modules
 
 .PHONY: test
 test:
-	$(GO) test $(GOTESTFLAGS) -mod=vendor -tags='sqlite sqlite_unlock_notify' $(GO_PACKAGES)
+	cd models/fixtures && \
```

Comment:
Just add another test i.e. `make test-prefix-tablename` to do that. And you should always define enviroment variables to change the behaviour.

Reply:
done